### PR TITLE
[Bing search] add action in bing search skill manifest

### PR DESCRIPTION
**Pull request by KayMKM**
### Purpose
*What is the context of this pull request? Why is it being done?*
add action in bing search skill manifest

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

File: skills/csharp/experimental/bingsearchskill/wwwroot/manifest/manifest-1.0.json [hunk 1]
```diff
@@ -26,6 +26,41 @@
     "message": {
       "type": "message",
       "description": "Receives the users utterance and attempts to resolve it using the skill's LU models"
+    },
+    "bingSearch": {
+      "description": "Search the result of keyword in Bing",
```
Comment (darrenj) [inline skills/csharp/experimental/bingsearchskill/wwwroot/manifest/manifest-1.0.json]: Perform a Bing Search using a provided keyword.

File: skills/csharp/experimental/bingsearchskill/wwwroot/manifest/manifest-1.0.json [hunk 2]
```diff
@@ -26,6 +26,41 @@
...
+      "type": "event",
+      "name": "BingSearch",
+      "value": {
+        "$ref": "#/definitions/inputObject"
```
Comment (darrenj) [inline skills/csharp/experimental/bingsearchskill/wwwroot/manifest/manifest-1.0.json]: keywordSearchRequest ?

File: skills/csharp/experimental/bingsearchskill/wwwroot/manifest/manifest-1.0.json [hunk 3]
```diff
@@ -26,6 +26,41 @@
...
+      },
+      "resultValue": {
+        "$ref": "#/definitions/responseObject"
```
Comment (darrenj) [inline skills/csharp/experimental/bingsearchskill/wwwroot/manifest/manifest-1.0.json]: keywordSearchResponse?

**Review by darrenj — CHANGES_REQUESTED** [review]

File: skills/csharp/experimental/bingsearchskill/wwwroot/manifest/manifest-1.0.json (hunk 1 above)
Comment (KayMKM) [reply skills/csharp/experimental/bingsearchskill/wwwroot/manifest/manifest-1.0.json]: fixed

**Pull request closed without merging**